migrations/versions/40affbf3022b_add_filter_colume_in_webhooks.py

```diff
@@ -0,0 +1,84 @@
+"""add filter colume in webhooks
+
+Revision ID: 40affbf3022b
+Revises: 5d5f801f28e7
+Create Date: 2023-08-28 12:30:35.171176
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '40affbf3022b'
+down_revision = '5d5f801f28e7'
+branch_labels = None
+depends_on = None
+
+
+def upgrade() -> None:
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.drop_index('ix_agent_schedule_agent_id', table_name='agent_schedule')
+    op.drop_index('ix_agent_schedule_expiry_date', table_name='agent_schedule')
```

Comment:
remove all drop index

why is there alter_columns ?

if alter_columns were auto generated remove them also

Reply:
I just used these two commands:
docker compose exec -it backend bash
 then
alembic revision --autogenerate -m "add filter colume in webhooks"
and then this file was automatically generated.

Reply:
Made the changes as you suggested